bootstrap.sh

```
@@ -57,57 +81,48 @@ ensure_stow() {
   fi
 }
 
+# Run all executable install hooks matching the profile, e.g., install.d/common-*.sh
 run_profile_hooks() {
   local pattern="${HOOKS_DIR}/${PROFILE}-*.sh"
   echo "🚀 Running hooks for profile '$PROFILE'..."
-  for hook in $pattern; do
+  for hook in "$pattern"; do
```

Shellcheck (0.10.0)

[error] Since you double quoted this, it will not word split, and the loop will only run once.

(SC2066)